### ISSUE TITLE
Sudden Termination of Python when importing pinocchio.casadi

### ISSUE THREAD

**Issue by xeonz1** [stack-of-tasks/pinocchio#1578]
The Import failed without any message and python just exit. I can still compile c++ codes with pinocchio. What should I do to debug? It still works days ago.

**Comment by xeonz1** [comment]
pinocchio can be normally imported. only pinocchio.casadi is unavailable

**Comment by jcarpent** [comment]
On which commit are you working now? What was the former one?

**Comment by xeonz1** [comment]
> 

I debugged for a while and found that when import pinocchio.casadi a SX is created with value a tuple (0), which may caused exception. I guess it may use import casadi.casadi instead of import casadi

**Comment by jcarpent** [comment]
Sorry, but you are hard to follow. 
Could you provide a tiny example that is not working on your side in such a way we can try on our side?

**Comment by xeonz1** [comment]
> 

Just import pinocchio.casadi. I see Process finished with exit code -1073741819 (0xC0000005)

**Comment by jcarpent** [comment]
I'm using casadi 3.5.5 and everything works perfectly well.
If it was working well a few days ago, it should remain the case. I suspect some change on your side.

Please consider reinstalling a clean version in a clean environment.
